migrate_db.py

```diff
@@ -0,0 +1,64 @@
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import Column, Integer, ARRAY, BIGINT, create_engine, MetaData
+from contextlib import contextmanager
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
+from csv import DictReader
+
+from app.settings import URL, DATA_FILE, TABLE_NAME
+
+Base = declarative_base()
+
+
+class Query(Base):
+    __tablename__ = TABLE_NAME
+    id_ = Column('id_', Integer, primary_key=True)
+    query_id = Column("query_id", Integer)
+    status = Column("status", Integer)
+    time = Column("time", BIGINT)
+    rows = Column("rows", BIGINT)
+    threads = Column("threads", ARRAY(Integer), nullable=True)
+
+    def __init__(self, row, id_):
+    	self.id_ = id_
```

Comment:
I am pretty sure this will throw `IntendationError`, since Python is sensitive to indentations.

Reply:
Agree, missed it